src/schema/posts.ts

```diff
@@ -833,6 +852,25 @@ export const resolvers: IResolvers<any, Context> = {
       });
       return { _: true };
     },
+    submitExternalLink: async (
+      _,
+      {
+        sourceId,
+        url,
+        commentary,
+      }: { sourceId: string; url: string; commentary: string },
+      ctx,
+    ): Promise<GQLEmptyResponse> => {
+      await createExternalLink(
```

Comment:
Important thing that we missed, we need to validate permissions!

Reply:
Ah good spot!
Added the `Post` permissions now

Reply:
Can you please add relevant tests?

Reply:
@idoshamun Added relevant cases for permissions